### PR TITLE
Package owl.0.3.8

### PR DESCRIPTION
**Pull request by mor1**
### `owl.0.3.8`

Owl - An OCaml Numerical Library

Owl is an OCaml numerical library. It supports N-dimensional arrays, both dense and sparse matrix operations, linear algebra, regressions, fast Fourier transforms, and many advanced mathematical and statistical functions (such as Markov chain Monte Carlo methods). Recently, Owl has implemented algorithmic differentiation which essentially makes developing machine learning and neural network algorithms trivial.

---
* Homepage: https://github.com/owlbarn/owl
* Source repo: git+https://github.com/owlbarn/owl.git
* Bug tracker: https://github.com/owlbarn/owl/issues

---

---
### 0.3.8 (2018-05-22)

* Add initial support for dataframe functionality.
* Add IO module for Owl's specific file operations.
* Add more helper functions in Array module in Base.
* Add core functions such as one_hot, slide, and etc.
* Fix normalisation neuron in neural network module.
* Fix building, installation, and publishing on OPAM.
* Fix broadcasting issue in Algodiff module.
* Support negative axises in some ndarray functions.
* Add more statistical distribution functions.
* Add another higher level wrapper for CBLAS module.
:camel: Pull-request generated by opam-publish v0.3.5

### CODE REVIEW

**Comment by mor1** [comment]
Bump...?

The failure appears to be on Ubuntu 14.04 as before, due to platform library issues around lapacke, blas, etc. (I can't get into any more detail right now as ci.ocamllabs.io appears to be down for me.)

File: packages/owl-top/owl-top.0.3.8/opam [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+build: [
+  [ "jbuilder" "subst" "-n" name ] {pinned}
+  [ "jbuilder" "build" "-p" name "-j" jobs ]
+]
+
+depends: [
+  "jbuilder" {build}
+  "ocaml-compiler-libs"
+  "owl"
```
Comment (mseri) [inline packages/owl-top/owl-top.0.3.8/opam]: Does this require `{ >= "3.8.0" }`, similarly for `owl-zoo` and the other libraries in the opam files, or they are supposed to be all cbackwards ompatible?

Comment (mor1) [reply packages/owl-top/owl-top.0.3.8/opam]: Ah! Sorry, yes they probably should have that constraint in that case. I don't think they're intended to be backwards compatible (/cc @ryanrhymes).

I'll update the PR. Thanks!

Comment (ryanrhymes) [reply packages/owl-top/owl-top.0.3.8/opam]: No, they are not intended to be backwards compatible.

**Comment by mseri** [comment]
Thanks. I think to remember that these linking issues are not new. I'll merge, I try to spend some time to see if they are fixable

**Comment by mor1** [comment]
Thanks!

**Pull request merged**